scripts/ensure-python.sh

```diff
@@ -6,7 +6,7 @@ set -eu
 
 PY_VERSION=3.9.10
 
-if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+if [[ "$OSTYPE" == "linux-gnu"* || "$OSTYPE" == "darwin"* ]]; then
     which "/opt/circleci/.pyenv/versions/$PY_VERSION/bin/python" || pyenv install "$PY_VERSION"
```

Comment:
Is this check even necessary anymore? I mean, this condition should always be true since we are only testing on linux and macos.

This makes sense if the body of the `if` condition is really applicable to both OS types. Is `/opt/circleci/` correct on both?

Reply:
Great catch! https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/340/commits/f1c11ec557807f70d4ff6595a4238836f51c31eb